src/ProgressOnderwijsUtils/Data/ParameterizedSqlObjectMapper.cs

```diff
@@ -473,11 +522,10 @@ static ByMetaObjectImpl()
                 {
                     var writablePropCount = 0;
                     foreach (var mp in metadata) { //perf:no LINQ
-                        if (mp.CanWrite && SupportsType(mp.DataType)) {
+                        if (mp.CanWrite && (IsSimpletype(mp) || IsCreatableType(mp))) {

```

Comment:
Ik vermoed dat `(IsSimpletype(mp) || IsCreatableType(mp)` een eigen method verdient (en het is me niet zo duidelijk dat die andere twee hoeven bestaan)

Reply:
Die IsSimpleType wordt nog aangeroepen vanuit IsCreatableType